app/Domain/Clients/Templates/delClient.blade.php

```diff
@@ -12,7 +12,7 @@
     <div class="pageicon"><span class="fa fa-address-book"></span></div>
     <div class="pagetitle">
         <h5>{{ __("label.administration") }}</h5>
-        <h1><?php echo sprintf($tpl->__('headline.delete_client'), $client['name']); ?></h1>
+        <h1><?php echo sprintf($tpl->__('headline.delete_client'), $client->name); ?></h1>
```

Comment:
Can you replace this with a blade syntax? `{!! sprintf( __('headline.delete_client'), $tpl->escape($client->name)) !!}`